src/swapd/swap_state.rs

```diff
@@ -548,6 +556,35 @@ impl State {
             _ => unreachable!("checked state"),
         }
     }
+
+    /// Update Bob Reveal with Alice's reveal
+    pub fn b_sup_alice_reveal(&mut self, reveal: Reveal) -> bool {
```

Comment:
I'm asking here because otherwise I don't ask ;p

What's the meaning of `.._sup_..` in the function name? I get the `a|b_` for Alice and Bob but not the sup.

Reply:
I don't know either